src/errors/test-run/templates.js

```diff
@@ -352,4 +352,8 @@ export default {
     [TEST_RUN_ERRORS.allowMultipleWindowsOptionIsNotSpecifiedError]: err => `
         You should activate multi window mode (enable the "allow-multiple-windows" run option) to use the "${err.methodName}" method.
     `,
+
+    [TEST_RUN_ERRORS.parentWindowNotFoundError]: () => `
+        Cannot find the parent window.
```

Comment:
Can we extend this error message with steps on how to fix the problem? 
Something like this:
```js
Cannot find the parent window. Make sure that the current window was opened from another window.... 
```

Reply:
We can. Need to wait for @VasilyStrelyaev